r2/r2/public/static/js/scrollupdater.js

```diff
@@ -32,8 +32,8 @@
         },
 
         _listen: function() {
-            var throttledUpdate = _.throttle($.proxy(this, '_updateThings'), 20)
```

Comment:
@ajacksified (Since you commited this code)
The `wait` value may have been intended to be `20 * 1000`, since that is what is used in `TimeText` (https://github.com/reddit/reddit/blob/da549027955c28b2e99098a22c814fc2ba729e11/r2/r2/public/static/js/timetext.js#L29)